frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx

```diff
@@ -0,0 +1,84 @@
+import { t } from "ttag";
+
+import * as Lib from "metabase-lib";
+
+import { AddAggregationButton } from "../AddAggregationButton";
+import { AggregationItem } from "../AggregationItem";
+import { BreakoutColumnList } from "../BreakoutColumnList";
+import {
+  AggregationsContainer,
+  ColumnListContainer,
+  SectionTitle,
+} from "../SummarizeSidebar.styled";
+
+import { STAGE_INDEX } from "./use-summarize-query";
+
+export type SummarizeContentProps = {
+  query: Lib.Query;
+  aggregations: Lib.AggregationClause[];
+  hasAggregations: boolean;
+  handleAddAggregations: (aggregations: Lib.Aggregable[]) => void;
+  handleUpdateAggregation: (
+    aggregation: Lib.AggregationClause,
+    nextAggregation: Lib.Aggregable,
+  ) => void;
+  handleRemoveAggregation: (aggregation: Lib.AggregationClause) => void;
+  handleAddBreakout: (column: Lib.ColumnMetadata) => void;
+  handleUpdateBreakout: (
+    clause: Lib.BreakoutClause,
+    column: Lib.ColumnMetadata,
+  ) => void;
+  handleRemoveBreakout: (column: Lib.ColumnMetadata) => void;
+  handleReplaceBreakouts: (column: Lib.ColumnMetadata) => void;
```

Comment:
nit: per convention `handle` is used for the handler to pass to the prop, while `on` is the prefix of the prop itself, e.g. `onRemoveBreakout={handleRemoveBreakout}`. can we rename these handle* props to on*?